vrecord

```diff
@@ -1298,8 +1318,15 @@ if [[ "${VIDEO_CODEC_CHOICE}" = "FFV1" ]] ; then
 fi
 _review_option "AUDIO_CODEC_CHOICE" "Which audio codec?" "${AUDIO_CODEC_OPTIONS[@]}"
 
+# Check for user suffix to override automatic settings
+
+if [[ -n "${USER_SUFFIX}" ]] ; then
+    SUFFIX="${USER_SUFFIX}"
```

Comment:
hmm, then there's no way to disable the suffix. Perhaps it's worth removing the auto-suffixing and then in _verify_conf_items add
```
if [[ "$(grep -c "${CONF_VARIABLE}" "${CONF_FILE}")" = "0" ]] ; then
    if [[ "${CONF_VARIABLE}" = "USER_SUFFIX" ]] ; then
        _report -w "Please note that moving forward, vrecord will not automatically add suffixes. If you'd like a suffix, please indicate it in the new section of the preferences.
    fi
   echo "${CONF_VARIABLE}=\"\"" >> "${CONF_FILE}"
fi
```

and then call `_verify_conf_items "USER_SUFFIX" "${CONFIG_FILE}"` later in the script